From 86622a025435aaaf7df5cc39c3310018b8cb4b89 Mon Sep 17 00:00:00 2001
From: Ievgen Mutavchi <Ievgen_Mutavchi@comcast.com>
Date: Wed, 7 Apr 2021 14:08:12 -0400
Subject: [PATCH] Reduce BrowserConsoleLog

---
 WebKitBrowser/WebKitImplementation.cpp | 11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

Index: git/WebKitBrowser/WebKitImplementation.cpp
===================================================================
--- git.orig/WebKitBrowser/WebKitImplementation.cpp
+++ git/WebKitBrowser/WebKitImplementation.cpp
@@ -487,7 +487,11 @@ static GSourceFuncs _handlerIntervention
                 , TCPKeepAlive(false)
                 , ClientCert()
                 , ClientCertKey()
+#ifdef WEBKIT_GLIB_API
+                , LogToSystemConsoleEnabled(true)
+#else
                 , LogToSystemConsoleEnabled(false)
+#endif
                 , WatchDogCheckTimeoutInSeconds(0)
                 , WatchDogHangThresholdInSeconds(0)
                 , LoadBlankPageOnSuspendEnabled(false)
@@ -2955,9 +2959,17 @@ static GSourceFuncs _handlerIntervention
         exit(1);
     }
 
-    /* static */ void willAddDetailedMessageToConsole(WKPageRef, WKStringRef source, WKStringRef, uint64_t line, uint64_t column, WKStringRef message, WKStringRef, const void* clientInfo)
+    /* static */ void willAddDetailedMessageToConsole(WKPageRef, WKStringRef source, WKStringRef, uint64_t line, uint64_t column, WKStringRef message, WKStringRef url, const void* clientInfo)
     {
         auto &self = *const_cast<WebKitImplementation*>(static_cast<const WebKitImplementation*>(clientInfo));
+        #if 1
+        if (WPEFramework::Trace::TraceType<BrowserConsoleLog, &WPEFramework::Core::System::MODULE_NAME>::IsEnabled() == false)
+            return;
+        string urlStr = WebKit::Utils::WKStringToString(url);
+        string messageStr = WebKit::Utils::WKStringToString(message);
+        fprintf(stderr, "[%s]:%s:%llu,%llu %s\n", self.GetConsoleLogPrefix().c_str(), Core::FileNameOnly(urlStr.c_str()), line, column, messageStr.c_str());
+        return;
+        #endif
         TRACE_GLOBAL(BrowserConsoleLog, (self.GetConsoleLogPrefix(), message, line, column));
     }
 #endif // !WEBKIT_GLIB_API
Index: git/WebKitBrowser/Extension/main.cpp
===================================================================
--- git.orig/WebKitBrowser/Extension/main.cpp
+++ git/WebKitBrowser/Extension/main.cpp
@@ -171,7 +171,10 @@ private:
     {
         string messageString = Core::ToString(webkit_console_message_get_text(message));
         uint64_t line = static_cast<uint64_t>(webkit_console_message_get_line(message));
-
+#if 1
+        fprintf(stderr, "[%s]:%llu %s\n", host->_consoleLogPrefix.c_str(), line, messageString.c_str());
+        return;
+#endif
         TRACE_GLOBAL(BrowserConsoleLog, (host->_consoleLogPrefix, messageString, line, 0));
     }
     static gboolean userMessageReceivedCallback(WebKitWebPage* page, WebKitUserMessage* message)
